### ISSUE TITLE
`i18nextInit` is not defined

### ISSUE THREAD

**Issue by aricooperdavis** [LibreScore/dl-librescore#8]
When I use the "Bookmark" method the buttons load but do not do anything, and the following error appears in the console:

```js
Uncaught (in promise) ReferenceError: i18nextInit is not defined
    <anonymous> https://github.com/LibreScore/dl-librescore/releases/latest/download/dl-librescore.user.js:3981
    <anonymous> https://github.com/LibreScore/dl-librescore/releases/latest/download/dl-librescore.user.js:3982
    <anonymous> https://github.com/LibreScore/dl-librescore/releases/latest/download/dl-librescore.user.js:35163
dl-librescore.user.js:3981:11
```

**Comment by PeterNjeim** [comment]
Cannot reproduce. The error you shared is not relevant to the downloading issue you're facing. I get the same console error but the buttons do indeed work.

Please share what browser you're using and its version number

**Comment by aricooperdavis** [comment]
Ahhh, my mistake - once I'd disabled my privacy extensions the buttons do work despite the error in the console. Sorry!

**Comment by Xmader** [comment]
This was actually an error which was addressed in https://github.com/LibreScore/dl-librescore/commit/966a8b4620c1bae6675508b4fc0280579b059a2a

**Comment by PeterNjeim** [comment]
Why was this re-opened? This issue is about the download buttons not working, the console error is unrelated

**Comment by Xmader** [comment]
unrelated but worth fixing because it causes another error that [entries in Tempermonkey menu](https://www.tampermonkey.net/documentation.php#api:GM_registerMenuCommand) couldn't be properly displayed

**Comment by PeterNjeim** [comment]
I fixed the console error in the commit you linked, menu is working fine for me

**Comment by PeterNjeim** [comment]
fixed in f2cc1ae7b83fbe8ae08d676655685cd3ee72a159..1b9d0ac55e0bb4aa4c201cd288df8d36b7442641
